python/whylogs/api/writer/whylabs.py

```diff
@@ -174,8 +177,43 @@ def option(  # type: ignore
             self._ssl_ca_cert = ssl_ca_cert
         return self
 
+    def write_feature_weights(self, file: FeatureWeight, **kwargs: Any) -> Tuple[bool, str]:
+        """Put feature weights for the specified dataset.
+
+        Parameters
+        ----------
+        file : FeatureWeight
+            FeatureWeight object representing the Feature Weights for the specified dataset
+
+        Returns
+        -------
+        Tuple[bool, str]
+            Tuple with a boolean (1-success, 0-fail) and string with the request's status code.
+        """
+        self._feature_weights = file.to_dict()
+        if kwargs.get("dataset_id") is not None:
+            self._dataset_id = kwargs.get("dataset_id")
+        return self._do_upload_feature_weights()
+
+    def get_feature_weights(self, **kwargs: Any) -> FeatureWeightResponse:
```

Comment:
suggest we return a `FeatureWeights` here.

Also we might reconsider the name of this class now that its not only writing (outside this PR).

Reply:
We might need another class, because the GET request will actually return a list of  `FeatureWeights`, along with metadata information for the whole list.

Maybe making another class, like `FeatureWeightsSet` or something?

Reply:
Is it because of segments? Why is the GET returning more than one `FeatureWeights`?

Reply:
Maybe the easiest thing to do for now is to return FeatureWeights by retrieving the first element of the list and, once segmented feature weights are supported, revisit this. What do you think @jamie256 ?

Reply:
With the last commit this is being done: get_feature_weights is retrieving the first element of the list. If we stick to it, we should revisit this once segmented feature weights is supported.

Reply:
Sounds good, its probably just returning a dictionary of these keyed by segment then.